modules/custom/activity_creator/src/ActivityFactory.php

```diff
@@ -559,20 +563,16 @@ protected function processArguments(array $arguments, array $output, Message $me
    *
    * @param array $output
    *   The templated text to be replaced.
-   * @param bool $clear
-   *   Determine if unused token should be cleared.
+   * @param array $options
+   *   Token options.
```

Comment:
@volodymyr-sydor just thought that it would be good to add more detailed explanation, for example: 
> Token options, like 'langcode' or 'clear' flag that determine if unused token should be cleared.

Reply:
@ribel 
Yes, we can extend it, but when we have more parameters then we should describe each of them
So, that's why I guess we should use a more general definition, like the array of options that can be changed/extended at any time.